cpp/src/slice2matlab/Main.cpp

```diff
@@ -358,30 +107,13 @@ namespace
         // The class directory is useful if you want to add additional supporting files for the class. We only
         // generate a single file for a class so we use option 2.
         //
-        const string cls = lookupKwd(v[v.size() - 1]);
+        const string cls = v[v.size() - 1];
         path += cls + ".m";
 
         out.open(path);
         FileTracker::instance()->addFile(path);
     }
 
-    //
```

Comment:
This function was replaced by `cont->getMappedScoped(".").substr(1)`.
And just to re-iterate, I plan to cleanup all these leading/trailing separators after I finish the `xxx:identifiers`.